visualization/app/codeCharta/state/selectors/allNecessaryRenderDataAvailable/utils/areDynamicSettingsAvailable.spec.ts

```diff
@@ -2,9 +2,9 @@ import { ColorMode, DynamicSettings, SortingOption } from "../../../../codeChart
 import { areDynamicSettingsAvailable, _isDynamicSettingAvailable } from "./areDynamicSettingsAvailable"
 
 describe("areDynamicSettingsAvailable", () => {
-	it("should return false when recent files aren't set", () => {
+	it("should return false when colorRange aren't set", () => {
 		const dynamicSettings = {
-			recentFiles: null
+			colorRange: null
```

Comment:
Is the method name of this test still applicable after recent files are deleted? Is the colour range really that critical to the dynamic settings?

Reply:
good question. This is just an example action and not optimal. But better as it was before, when I introduced it :)

Reply:
Very fair point, thanks for the explanation.
LGTM! 🚀